README.md

```diff
@@ -333,6 +333,7 @@ Name | Website | Region
 [reinteractive](/company-profiles/reinteractive.md) ⚠️️ | https://reinteractive.net/ |
 [RenoFi](/company-profiles/renofi.md) | https://renofi.com/ |
 [Research Square](/company-profiles/research-square.md) | https://www.researchsquare.com/ | USA
+[Retail Zipline](/company-profiles/retail-zipline.md) | https://www.retailzipline.com.com/ | USA, Canada, UK, Mexico
```

Comment:
Hi @jhubert, thanks for the PR.

Here, does "USA, Canada, UK, Mexico" mean that you only hire employees from these countries? This is what this field is intended to mean.